book/routing.rst

```diff
@@ -899,22 +913,27 @@ accomplished with the following route configuration:
 
         return $collection;
 
-.. versionadded:: 2.2
-    The ``methods`` option was introduced in Symfony 2.2. Use the ``_method``
-    requirement in older versions.
-
 Despite the fact that these two routes have identical paths
-(``/api/posts/{id}``), the first route will match only GET or HEAD requests and
-the second route will match only PUT requests. This means that you can display
-and edit the post with the same URL, while using distinct controllers for the
-two actions.
+(``/api/posts/{id}``), the first route will match only GET or HEAD
+requests and the second route will match only PUT requests. This means
+that you can display and edit the post with the same URL, while using
+distinct controllers for the two actions.
 
 .. note::
 
-    If no ``methods`` are specified, the route will match on *all* methods.
+    If no ``@Method`` or ``methods`` parameter is specified, the route
+    will match on *all* methods.
 
-Adding a Host Requirement
-~~~~~~~~~~~~~~~~~~~~~~~~~
+.. seealso::
+
+    Unfortunately, life isn't quite this simple, since most browsers do not
+    support sending all method attributes in an HTML form. Fortunately,
+    Symfony provides you with a simple way of working around this limitation.
+    To learn about it read cookbook article
+    :doc:`/cookbook/routing/method_parameters`
```

Comment:
I would say something like:

>
Most *browsers* only support GET and POST requests. To build a form that uses a different
HTTP method (e.g. PUT), see <link>.